compliance/http/src/test/java/org/eclipse/rdf4j/http/server/ProtocolTest.java

```diff
@@ -482,6 +516,62 @@ public void run() {
 		threadPool.shutdownNow();
 	}
 
+	private Set<Namespace> getNamespaces(String location)
+		throws Exception
+	{
+		URL url = new URL(location);
+		HttpURLConnection conn = (HttpURLConnection)url.openConnection();
+		try {
+			conn.setRequestProperty("Accept", "text/csv; charset=utf-8");
+			conn.connect();
+
+			InputStream connIs = conn.getInputStream();
+			try {
+				int responseCode = conn.getResponseCode();
+				if (responseCode != HttpURLConnection.HTTP_OK) {
+					String response = "location " + location + " responded: " + conn.getResponseMessage()
+							+ " (" + responseCode + ")";
+					fail(response);
+				}
+
+				String responseText = IOUtil.readString(connIs);
+				String[] rows = responseText.split("\\r?\\n");
```

Comment:
Looking at it further using 

```
mvn dependency:tree
```

You may need to add a runtime/test dependency to opencsv and/or queryresultio-text to get my suggestion to work as http-compliance only seems to link to queryresultio-binary, queryresultio-sparqljson and queryresultio-sparqlxml right now.